src/plugins/poster/poster.js

```diff
@@ -95,12 +95,15 @@ export default class PosterPlugin extends UIContainerPlugin {
   }
 
   clicked() {
-    if (!this.options.chromeless || this.options.allowUserInteraction) {
-      this.playRequested = true
-      this.update()
-      this.container.play()
+    // Let "click_to_pause" plugin handle click event if media has started playing
+    if (! this.hasStartedPlaying) {
```

Comment:
can't we inline this condition?
```js
if (! this.hasStartedPlaying && (!this.options.chromeless || this.options.allowUserInteraction)) {}
```

Reply:
The condition could be rewrited, maybe removing the return false because of hasStartedPlaying check. I writed it this way to ensure it do not break anything.

Reply:
Also if you rewrite the condition that way, with audio only, if you seek almost to end, then wai for video to stop and display poster play icon, then if you click, it does not start the video.

Reply:
@leandromoreira  In fact, the return false is required when click is handled by poster. Returning false disrupt the click event and container click if not triggered. Otherwise, if triggered the media is immediatly paused by `click_to_pause` plugin.

Just experiment the branch by yourself :)